src/BenchmarkDotNet.Core/Exporters/Xml/IXmlSerializer.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace BenchmarkDotNet.Exporters.Xml
+{
+    public interface IXmlSerializer : IFluentXmlSerializer
```

Comment:
I would expect that every  `IFluentXmlSerializer` is `IXmlSerializer` and not every `IXmlSerializer` is `IFluentXmlSerializer`.

do we need to have two interfaces here?